benchmarks/Exporter/JaegerExporterBenchmarks.cs

```diff
@@ -33,18 +35,46 @@ namespace Benchmarks.Exporter
 #endif
     public class JaegerExporterBenchmarks
     {
+        private readonly byte[] Buffer = new byte[1024];
+
         [Params(1, 10, 100)]
         public int NumberOfBatches { get; set; }
 
         [Params(10000)]
         public int NumberOfSpans { get; set; }
 
+        private UdpClient jaegerServer;
+        private JaegerUdpClient jaegerClient;
         private SpanData testSpan;
 
         [GlobalSetup]
         public void GlobalSetup()
         {
             this.testSpan = this.CreateTestSpan();
+
+            this.jaegerServer = new UdpClient(new IPEndPoint(IPAddress.Any, 10018));
+            ThreadPool.QueueUserWorkItem(this.ReceivedData);
+
+            this.jaegerClient = new JaegerUdpClient();
+            this.jaegerClient.Connect("localhost", 10018);
```

Comment:
This looks to require a local Jaeger instance to be running on port 10018. Please can you add a README for the benchmark project with instructions on how to to get / setup Jaeger?

Reply:
If you look up a couple lines the "server" is spun up right before the client. It's not a real Jaeger instance, just a dummy UDP listener that does nothing with the packets. I'm basically testing the transmission mechanisms, not the payloads.

Reply:
Doh -I missed that, thanks.